src/main/java/frc/robot/ComputerMain.java

```diff
@@ -1,16 +1,36 @@
 package frc.robot;
 
+import frc.utils.CMDHandler;
+import frc.utils.controllers.keyboard.KeyboardController;
+
+import javax.swing.*;
+import java.nio.file.Path;
+
 /**
  * Unless you know what you are doing, do not rename this file because it's being used elsewhere.
  */
 public class ComputerMain {
 
-	public static void main(String[] args) {
-		startComputerPrograms();
-	}
+    public static void main(String... args) {
+        startComputerPrograms(args);
```

Comment:
```suggestion
        startComputerPrograms(args[0]);
```

Reply:
future computer programs may need arugments too

Reply:
their not, and if they do you wont give it args like that